api/api/clubs/tests.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.test import APITestCase
+
+from api.clubs.models import Club
+
+
+class ListClubViewTest(APITestCase):
+    """Tests ListClubView for getting the list of clubs."""
+
+    def test_listclubview_no_club_exist_return_empty_list(self):
```

Comment:
Naming of the testcase methods:
1. Start with `test{featureBeingTested}
2. Multiple words in a sentence in a camelCase.
3. Separate sentences by an underscore.

So this becomes:
`testListClubs_noClubExistsInDb_returnsEmptyList`